core/sql/src/main/java/org/trafodion/sql/TrafRegionStats.java

```diff
@@ -128,5 +155,90 @@ public TrafRegionStats (HTable table, Admin admin) throws IOException {
             }
             
     }
+
+
+    public TrafRegionStats () throws IOException {
+        Configuration config = HBaseConfiguration.create();
+
+        hbAdmin = new HBaseAdmin(config);
+    }
+
+    public boolean Open () throws IOException {
+        
+        clusterStatus = hbAdmin.getClusterStatus();
```

Comment:
HBaseAdmin is deprecated and its use is avoided. Please change it to use Admin instead. You need to pass the connection object and then use connection.getAdmin()